src/components/BigNumberPad.js

```diff
@@ -32,74 +32,68 @@ const padNumbers = [
     ['.', '0', '<'],
 ];
 
-class BigNumberPad extends React.PureComponent {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            timer: null,
-        };
-    }
+const BigNumberPad = ({numberPressed, longPressHandlerStateChanged, nativeID, toLocaleDigit}) => {
+    const [timer, setTimer] = useState(null);
 
     /**
      * Handle long press key on number pad.
      * Only handles the '<' key and starts the continuous input timer.
      *
      * @param {String} key
      */
-    handleLongPress(key) {
-        // Only handles deleting.
-        if (key !== '<') {
-            return;
-        }
-        this.props.longPressHandlerStateChanged(true);
-        const timer = setInterval(() => {
-            this.props.numberPressed(key);
-        }, 100);
-        this.setState({timer});
-    }
+    const handleLongPress = useCallback(
```

Comment:
I don't think that we need `useCallback` in this case since the computation is cheap and we're recreating as a lambda on render anyways, so the child component would still re-render even if we cached this function